services/core/shared/core/compat/sdk_v5/dynamicFees.js

```diff
@@ -142,10 +144,13 @@ const getEstimateFeeByteForBlock = async (blockBatch, innerPrevFeeEstPerByte) =>
 
 const getEstimateFeeByteForBatch = async (fromHeight, toHeight, cacheKey) => {
 	const transactionsDB = await getTransactionsIndex();
+	const { genesisHeight } = config;
+	const { defaultStartBlockHeight } = config.feeEstimates;
 
 	// Check if the starting height is permitted by config or adjust acc.
-	fromHeight = config.feeEstimates.defaultStartBlockHeight > fromHeight
-		? config.feeEstimates.defaultStartBlockHeight : fromHeight;
+	// Use incrementation to skip the genesis block - it is not needed
+	fromHeight = Math.max(...[defaultStartBlockHeight, genesisHeight + 1, fromHeight]
+		.filter(n => !Number.isNaN(n)));
```

Comment:
Kindly consider moving the logic to the shared layer instead, so that it is available for all SDK versions.

Reply:
That logic doesn't seem to be specific to all SDK versions. Can you explain how would you like to apply that remark?
Another thing, how do you test the patch against the SDKv4 when there is no migrator for prior versions?

Reply:
Please ignore the comment. The change seems unnecessary.